trustmanager/keystore.go

```diff
@@ -40,7 +42,47 @@ func NewKeyFileStore(baseDir string, p notary.PassRetriever) (*GenericKeyStore,
 	if err != nil {
 		return nil, err
 	}
-	return NewGenericKeyStore(fileStore, p), nil
+	migrateTo0Dot4(fileStore)
```

Comment:
would be great to move this migration down a level so it happens in `store.NewPrivateKeyFileStorage` where it can be optimized to only walk the `root_keys` and `tuf_keys` directories.